modulus/models/afno/afno.py

```diff
@@ -401,14 +404,14 @@ class AFNO(Module):
 
     Parameters
     ----------
-    img_size : Tuple[int, int]
-        Input image dimensions (height, width)
+    inp_shape : List[int]
```

Comment:
Do we know the impact of this APi change?
I have a checkpoint for AFNO getting posted on NGC using the previous name. Will this break that?